pytest_pyvista/pytest_pyvista.py

```diff
@@ -8,6 +8,20 @@
 import pyvista
 
 
+class RegressionError(RuntimeError):
+    """Error when regression does not meet the criteria"""
+
+    def __init__(self, msg):
```

Comment:
Do we need these __init__ methods?  Can't we instead inherit the parent class method?

Reply:
Maybe you don't needed. I guess it is just an habit.